Author: Vincent Bernat <bernat@debian.org>
Last-Updated: Andreas Tille <tille@debian.org>  Tue, 08 Mar 2016 11:15:04 +0100
Description: Avoid displaying a lot of information when starting xrdp and sesman
 .
 Remark by Andreas Tille: The better way would be to use some "#ifdef VERBOSE"
 pragma and forward this upstream.

--- a/sesman/config.c
+++ b/sesman/config.c
@@ -168,13 +168,13 @@ config_read_globals(int file, struct con
     }
 
     /* showing read config */
-    g_printf("sesman config:\r\n");
+    /* g_printf("sesman config:\r\n");
     g_printf("\tListenAddress:            %s\r\n", cf->listen_address);
     g_printf("\tListenPort:               %s\r\n", cf->listen_port);
     g_printf("\tEnableUserWindowManager:  %i\r\n", cf->enable_user_wm);
     g_printf("\tUserWindowManager:        %s\r\n", cf->user_wm);
     g_printf("\tDefaultWindowManager:     %s\r\n", cf->default_wm);
-    g_printf("\tAuthFilePath:             %s\r\n", ((cf->auth_file_path) ? (cf->auth_file_path) : ("disabled")));
+    g_printf("\tAuthFilePath:             %s\r\n", ((cf->auth_file_path) ? (cf->auth_file_path) : ("disabled"))); */
 
     return 0;
 }
@@ -225,11 +225,11 @@ config_read_logging(int file, struct log
     lc->log_file=g_strdup("./sesman.log");
   }
 
-  g_printf("logging configuration:\r\n");
+  /* g_printf("logging configuration:\r\n");
   g_printf("\tLogFile:       %s\r\n",lc->log_file);
   g_printf("\tLogLevel:      %i\r\n", lc->log_level);
   g_printf("\tEnableSyslog:  %i\r\n", lc->enable_syslog);
-  g_printf("\tSyslogLevel:   %i\r\n", lc->syslog_level);
+  g_printf("\tSyslogLevel:   %i\r\n", lc->syslog_level); */
 
   return 0;
 }
@@ -293,7 +293,7 @@ config_read_security(int file, struct co
     }
 
     /* printing security config */
-    g_printf("security configuration:\r\n");
+    /* g_printf("security configuration:\r\n");
     g_printf("\tAllowRootLogin:       %i\r\n", sc->allow_root);
     g_printf("\tMaxLoginRetry:        %i\r\n", sc->login_retry);
     g_printf("\tAlwaysGroupCheck:     %i\r\n", sc->ts_always_group_check);
@@ -305,7 +305,7 @@ config_read_security(int file, struct co
     else
     {
         g_printf("\tNo TSUsersGroup defined\r\n");
-    }
+    } */
 
     if (sc->ts_admins_enable)
     {
@@ -370,12 +370,12 @@ config_read_sessions(int file, struct co
     }
 
     /* printing security config */
-    g_printf("session configuration:\r\n");
+    /* g_printf("session configuration:\r\n");
     g_printf("\tMaxSessions:                 %i\r\n", se->max_sessions);
     g_printf("\tX11DisplayOffset:            %i\r\n", se->x11_display_offset);
     g_printf("\tKillDisconnected:            %i\r\n", se->kill_disconnected);
     g_printf("\tIdleTimeLimit:               %i\r\n", se->max_idle_time);
-    g_printf("\tDisconnectedTimeLimit:       %i\r\n", se->max_idle_time);
+    g_printf("\tDisconnectedTimeLimit:       %i\r\n", se->max_idle_time); */
 
     return 0;
 }
@@ -400,12 +400,12 @@ config_read_rdp_params(int file, struct
     }
 
     /* printing security config */
-    g_printf("X11rdp parameters:\r\n");
+    /* g_printf("X11rdp parameters:\r\n");
 
     for (i = 0; i < cs->rdp_params->count; i++)
     {
         g_printf("\tParameter %02d                   %s\r\n", i, (char *)list_get_item(cs->rdp_params, i));
-    }
+    } */
 
     return 0;
 }
@@ -430,12 +430,12 @@ config_read_vnc_params(int file, struct
     }
 
     /* printing security config */
-    g_printf("Xvnc parameters:\r\n");
+    /* g_printf("Xvnc parameters:\r\n");
 
     for (i = 0; i < cs->vnc_params->count; i++)
     {
         g_printf("\tParameter %02d                   %s\r\n", i, (char *)list_get_item(cs->vnc_params, i));
-    }
+    } */
 
     return 0;
 }
--- a/sesman/sesman.c
+++ b/sesman/sesman.c
@@ -154,7 +154,7 @@ main(int argc, char **argv)
     if (1 == argc)
     {
         /* no options on command line. normal startup */
-        g_printf("starting sesman...\n");
+        /* g_printf("starting sesman...\n"); */
         daemon = 1;
     }
     else if ((2 == argc) && ((0 == g_strcasecmp(argv[1], "--nodaemon")) ||
--- a/xrdp/xrdp.c
+++ b/xrdp/xrdp.c
@@ -510,7 +510,7 @@ main(int argc, char **argv)
 
         if (0 != pid)
         {
-            g_writeln("process %d started ok", pid);
+            /* g_writeln("process %d started ok", pid); */
             /* exit, this is the main process */
             g_deinit();
             g_exit(0);
